rclcpp/test/rclcpp/executors/test_static_executor_entities_collector.cpp

```diff
@@ -190,6 +190,7 @@ TEST_F(TestStaticExecutorEntitiesCollector, add_remove_basic_node) {
 
   // Still one for the executor
   EXPECT_EQ(1u, entities_collector_->get_number_of_waitables());
+  entities_collector_->fini();
```

Comment:
Same question for all these `fini()` calls. Should they go inside a `SCOPE_EXIT` block?

Reply:
updated